index.js

```diff
@@ -1,9 +1,15 @@
+const path = require('path');
+const fs = require('fs');
 const nunjucks = require('nunjucks');
 const arc = require('@architect/functions');
 const static = arc.http.helpers.static;
-const mapping = require('@architect/shared/assets.json');
-const path = require('path');
-const fs = require('fs');
+
+// arc-rapptor also uses the SHARED_PATH env variable:
+const assetPath = process.env.SHARED_PATH ? `${process.env.SHARED_PATH}/assets.json` : '@architect/shared/assets.json';
```

Comment:
Who sets this?

Reply:
Was trying to make this consistent with what we're doing in https://github.com/firstandthird/arc-rapptor/blob/master/index.js#L5, but can take the SHARED_PATH part out if you don't think it's an issue.

Reply:
makes sense